packages/base-driver/lib/jsonwp-proxy/protocol-converter.js

```diff
@@ -1,317 +0,0 @@
-import _ from 'lodash';
```

Comment:
unfortunately we cannot delete this proxy as there are still some subdrivers, like windows one or ios-webview client that use JWP protocol. The first one is just abandoned by MS and the second one requires major atom updates to fully support w3c